test/functional/sync_plan/list_test.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require File.join(File.dirname(__FILE__), '../sync_plan/sync_plan_helpers')
+require File.join(File.dirname(__FILE__), '../organization/organization_helpers')
+
+module HammerCLIKatello
+  describe SyncPlan::ListCommand do
+    include SyncPlanHelpers
+
+    it 'allows minimal options' do
+      expect_sync_plan_search(1, nil, nil)
+
+      run_cmd(%w(sync-plan list --organization-id 1))
+    end
+  end
+
+  describe 'organization ID and LABEL can be resolved from organization' do
```

Comment:
Just the organization ID is resolved, not the label. I'd change this description.